jupyterhub/proxy.py

```diff
@@ -466,7 +476,21 @@ def _default_semaphore(self):
     def _concurrency_changed(self, change):
         self.semaphore = asyncio.BoundedSemaphore(change.new)
 
+    # https://github.com/jupyterhub/configurable-http-proxy/blob/4.5.1/bin/configurable-http-proxy#L92
+    log_level = Enum(
+        ["debug", "info", "warn", "error", "DEBUG", "INFO", "WARN", "ERROR"],
```

Comment:
https://github.com/jupyterhub/configurable-http-proxy/blob/4.5.1/bin/configurable-http-proxy#L92 says `debug, info, warn, error` but upper-case (as used by Python logging) also seems to work.

Reply:
Can use [CaselessStrEnum](https://traitlets.readthedocs.io/en/stable/trait_types.html#traitlets.CaselessStrEnum) to avoid repetition

Reply:
Thanks, I'm constantly amazed by all the hidden features in Traitlets :smiley: